### DISCUSSION TITLE
Stereo field collapsing with Beta 10

### DISCUSSION THREAD

**Discussion by Gregmang** [apohl79/audiogridder#972] [Help]
I thought I had beta 10 working flawlessly both locally and on an Ethernet connected laptop. Old projects opened just fine, connected to AG as normal.  But then I added a compressor (I think it was one one of TimP Nebula compressors - however replicated the same behavior with others) and I lost the stereo field and it sounded like everything collapsed to mono.  I tried again and specified dual-mono and no change - it was mono.  However this is odd - I then instantiated another plugin and chose stereo and it was like there was no processing. Audio was passing but there wasn’t any processing.

And check this out - projects opened with existing AG beta 10 instances worked fine - no issues.  The issue only showed itself when I added new plug-ins.

I deleted and rescanned both caches (local and remote system)
Deleted and reinstalled AG – issue remained.

So I reverted to 1.1.1 and everything is working fine.

**Comment by CiriusGodlover** [comment]
the same> acustica plugins sounds mono or no sound at all
